### PR TITLE
To be able to set Sub and Pub decorators together for same method

### PR DESCRIPTION
**Pull request by bintoll**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/nest/blob/master/CONTRIBUTING.md
- [-] Tests for the changes have been added (for bug fixes / features)
- [-] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?
This PR address this issue, that was initially evaluated as feature. https://github.com/flamewow/nestjs-asyncapi/issues/465

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: 465

## What is the new behavior?
AsyncApiPub and AsyncApiSub decorators now creates method decorator with different meta keys 

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

**Comment by flamewow** [comment]
Hey @bintoll , thanks for the PR

testing workflows for some reason were not executed on PR.
If you check it locally via `npm run test:e2e` you should get the failed tests, please fix

**Comment by bintoll** [comment]
@flamewow Hello! Done

File: lib/decorators/asyncapi-operation.decorator.ts [hunk 1]
```diff
@@ -38,8 +38,9 @@ function makeMessage(
   };
 }
 
-export function AsyncApiOperation(
-  ...options: AsyncApiOperationOptions[]
+export function AsyncApiOperationForMetaKey(
```
Comment (flamewow) [inline lib/decorators/asyncapi-operation.decorator.ts:41]: let's move that decorator to a separate file

Comment (flamewow) [reply lib/decorators/asyncapi-operation.decorator.ts:41]: lib/decorators/asyncapi-operation-for...

Comment (MaksimKiselev) [reply lib/decorators/asyncapi-operation.decorator.ts:41]: Hey guys. Need this feature. I've moved function to separate file, modify sample and tests for it:
https://github.com/flamewow/nestjs-asyncapi/pull/473

**Comment by flamewow** [comment]
closing in favor of https://github.com/flamewow/nestjs-asyncapi/pull/473

**Pull request closed without merging**